cumulusci/tasks/metadata_etl/value_sets.py

```diff
@@ -6,15 +6,20 @@
 class AddValueSetEntries(MetadataSingleEntityTransformTask):
     entity = "StandardValueSet"
     task_options = {
+        **MetadataSingleEntityTransformTask.task_options,
         "entries": {
             "description": "Array of standardValues to insert. "
             "Each standardValue should contain the keys 'fullName', the API name of the entry, "
             "and 'label', the user-facing label. OpportunityStage entries require the additional "
             "keys 'closed', 'won', 'forecastCategory', and 'probability'; CaseStatus entries "
             "require 'closed'.",
-            "required": False,
+            "required": True,
+        },
+        "api_names": {
+            "description": "List of API names of StandardValuesets to affect, "
+            "such as 'OpportunityStage', 'AccountType', 'CaseStatus'",
+            "required": True,
```

Comment:
n.b. the original reason `api_names` was not required at the top level is that leaving it blank is equivalent to wildcard (`*`). Wildcard is not meaningful for this task. Thanks for fixing.